### DISCUSSION TITLE
$null Variable

### DISCUSSION THREAD

**Discussion by TuArHH** [TobiasPSP/OReilly#13] [Fragen] [answered]
Hallo zusammen, 

häufig sehe ich das variablen bei initialisieren die null-variable verwendet wird oder das eine Funktion einer null-variable zugewiesen wird. Hat es einen Grund?  Was die $null ist weiß ich nur bei diesen zwei Punkten habe ich noch Wissenslücke

$null= .........

$null= Get-Function. 

Kann mir hier jemand auf die Sprünge helfen. 

Vielen Dank im Vorraus

**Comment by TobiasPSP** [comment, marked as answer]
Man verwendet dies, wenn der Rückgabewert einer Funktion nicht gebraucht wird. Ohne die Zuweisung an $null würde das Resultat in der Konsole sichtbar.

**Reply by TuArHH** [reply]
Vielen Dank Tobias. 

Habe ich späte im Buch auch erfahren ^^. War da noch nicht soweit.
